src/FacadeAccountCreation/FacadeAccountCreation.Core/Services/ComplianceScheme/ComplianceSchemeService.cs

```diff
@@ -293,4 +296,30 @@ public async Task<HttpResponseMessage> GetAllReasonsForRemovalsAsync()
             throw;
         }
     }
+
+    public async Task<List<ExportOrganisationSubsidiariesResponseModel>> ExportComplianceSchemeSubsidiaries(Guid userId, Guid organisationId, Guid complianceSchemeId)
+    {
+        HttpResponseMessage result = null;
+        var endpointConfigValue = $"{_config.GetSection("ComplianceSchemeEndpoints").GetSection("ExportComplianceSchemeSubsidiaries").Value}";
```

Comment:
The config needs to be added to app.config

    "ExportComplianceSchemeSubsidiaries": "api/compliance-schemes/{0}/schemes/{1}/export-subsidiaries"